flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/delegation/ParserImpl.java

```diff
@@ -64,6 +65,7 @@
     private final Supplier<CalciteParser> calciteParserSupplier;
     private final SqlExprToRexConverterFactory sqlExprToRexConverterFactory;
     private static final ExtendedParser EXTENDED_PARSER = ExtendedParser.INSTANCE;
+    private static final ExpressionParserImpl expressionParser = new ExpressionParserImpl();
```

Comment:
why this change? this deprecated parser has nothing to do with the SQL parser.

Reply:
Sorry, that's a leftover of when i tried to move the funcionality inside the `Parser` iface